ios/FluentUI/Table View/ListVnext.swift

```diff
@@ -0,0 +1,313 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import SwiftUI
+
+/// Pre-defined styles of icons
+@objc(MSFListIconVnextStyle)
+public enum MSFListIconVnextStyle: Int, CaseIterable {
+    case none
+    case iconOnly
+    case large
+}
+
+/// Pre-defined types of icons
+@objc(MSFListAccessoryType)
+public enum MSFListAccessoryType: Int, CaseIterable {
+    case none
+    case disclosure
+    case detailButton
+    case checkmark
+
+    var icon: UIImage? {
+        let icon: UIImage?
+        switch self {
+        case .none:
+            icon = nil
+        case .disclosure:
+            icon = UIImage.staticImageNamed("iOS-chevron-right-20x20")
+        case .detailButton:
+            icon = UIImage.staticImageNamed("more-24x24")
+        case .checkmark:
+            icon = UIImage.staticImageNamed("checkmark-24x24")
+        }
+        return icon
+    }
+}
+
+///Properties that make up cell content
+@objc(MSFListVnextCellData)
+public class MSFListVnextCellData: NSObject, ObservableObject, Identifiable {
+    public var id = UUID()
+    @objc @Published public var leadingIcon: UIImage?
+    @objc @Published public var title: String = ""
+    @objc @Published public var subtitle: String?
+    @objc @Published public var trailingIcon: MSFListAccessoryType = .none
+    @objc @Published public var titleLineLimit: Int = 1
+    @objc @Published public var subtitleLineLimit: Int = 1
+    @objc public var onTapAction: (() -> Void)?
+}
+
+///Properties that make up section content
+@objc(MSFListVnextSectionData)
+public class MSFListVnextSectionData: NSObject, ObservableObject, Identifiable {
+    public var id = UUID()
+    @objc @Published public var cells: [MSFListVnextCellData] = []
+    @objc @Published public var title: String?
+    @objc @Published public var layoutType: MSFListCellVnextLayoutType = .oneLine
+}
+
+///Properties that make up list content
+@objc(MSFListVnextState)
+public class MSFListVnextState: NSObject, ObservableObject {
+    @objc @Published public var hasBorder: Bool = false
+}
+
+@objc(MSFListCellVnextHeight)
+/// Pre-defined layout heights of cells
+public enum MSFListCellVnextLayoutType: Int, CaseIterable {
+    case oneLine
+    case twoLines
+    case threeLines
+
+    var height: CGFloat {
```

Comment:
Another thing that can be changed later (GitHub issue):
These values should be tokenized in the YAML file.
Something like:

```
AP_MSFListTokens:
  cellHeight: [ oneLine: 48, twoLines: 64, threeLines: 84 ]
  ...
```